olm_deploy/scripts/catalog-build.sh

```diff
@@ -16,5 +16,5 @@ if [ -n ${LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY} ] ; then
         echo "Unexpected message from oc port-forward: $PORT_FORWARD_STDOUT"
     fi
 fi
-echo "Pushing image ${IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY}"
-podman push  --tls-verify=false ${IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY}
+echo "Pushing image ${LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY}"
+podman push  --tls-verify=false ${LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY}
```

Comment:
This looks weird, we should always use `IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY` and use `LOCAL_IMAGE_ELASTICSEARCH_OPERATOR_REGISTRY` only when we set the former elsewhere.